packages/world/test/World.t.sol

```diff
@@ -230,6 +230,14 @@ contract WorldTest is Test, GasReporter {
     world.registerNamespace("test");
   }
 
+  function testTransferNamespace() public {
+    world.registerNamespace("testTransfer");
+    world.transferOwnership("testTransfer", address(1));
+    assertEq(NamespaceOwner.get(world, "testTransfer"), address(1), "new owner should be namespace owner");
+    // Expect the new owner to have access
+    assertEq(ResourceAccess.get(world, "testTransfer", address(1)), true, "new owner should have access");
```

Comment:
let's add a sanity check for ownership and access before the transfer, and an assertion for the old owner not having access anymore after the transfer

Reply:
Done. Note - I didn't add check for original ownership/permission on registration as its already covered in `testRegisterNamespace`.